docs/glossary.rst

```diff
@@ -0,0 +1,20 @@
+Glossary
+========
+
+.. glossary::
+
+   dashboard
+      Main page where you can see all your projects with their build status
+      and import a new project.
```

Comment:
Wonder if we should include a URL here `/dashboard/` to be explicit?

Reply:
Good idea, will do